src/components/Nav/components/NavAvatar/index.jsx

```diff
@@ -42,7 +42,6 @@ const GordonNavAvatar = ({ onLinkClick }) => {
   const buttonLink = forwardRef((props, ref) => (
     <Link
       {...props}
-      innerRef={ref}
       to={profile ? `/myprofile` : '/'}
```

Comment:
As above, we should just get rid of `forwardRef` here and use a plain old `Link`.